signalfx-forwarder-app/bin/eventstosfx.py

```diff
@@ -0,0 +1,171 @@
+# pylint: skip-file
```

Comment:
Sure this is desired?

Reply:
Yeah defintiely agree we want it but the lint is complaining that the code in tosfx and eventstosfx is duplicated and shoulld be refactored. I did tryi refactoring to a super class that inherits EventingCommand but it wasnt working and I ran out of time - hence just ignoring the lint.